cross-chain/solana/Anchor.toml

```diff
@@ -22,6 +22,7 @@ wallet = "~/.config/solana/id.json"
 
 [scripts]
 test = "npx ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
+authority = "ts-node --files ./typescript/transfer_authority.ts"
```

Comment:
This is interesting; I've never used the Anchor.toml file this way. I've usually put in my commands in a Makefile.

See the Makefile in this directory for more info.

Reply:
Moved to the Makefile. Pls see [3882cc4](https://github.com/keep-network/tbtc-v2/pull/688/commits/3882cc4e49a9240d7420f4d825bc4005a18dd4df)